ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -829,7 +847,7 @@ import '../nuxeo-button-styles.js';
     }
 
     _onCheckBoxTap(e) {
-      if (this.selectionEnabled) {
+      if (this.selectionEnabled && !this._isSelectAllActive) {
```

Comment:
Shouldn't this be
```suggestion
      if (this.selectionEnabled && !this.selectAllActive) {
```
?

Reply:
Same as discussed above.